flutter/assets/directory_asset_bundle.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/assets/directory_asset_bundle.h"
+
+#include <fcntl.h>
+#include <unistd.h>
+
+#include <utility>
+
+#include "glue/data_pipe_utils.h"
+#include "lib/ftl/files/path.h"
+
+namespace blink {
+
+void DirectoryAssetBundle::Create(
```

Comment:
There are other instances in this patch like this (the `ZipAssetService`). But I think that code was just moved and not newly authored. So not sure if we need to attend to it here.